Färdigställt första versionen

gyback opened the pull request:
Färdigställt den första versionen med fokus på att skriva stadgar, pm och vanliga dokument.
TODO Den är inte färdigställd för allapm då jag inte vet hur vi vill att den ska hanteras.
TODO Listmiljöer för Paragrafer kan behöva uppdateras.
TODO Ta ut motionsmallen och skapa ett eget dokument för det.

gyback committed:
Färdigställt första versionen

Färdigställt den första versionen med fokus på att skriva stadgar,pm och vanliga dokument.
TODO Den är inte färdigställd för allapm då jag inte vet hur vi vill att den ska hanteras.
TODO Listmiljöer för Paragrafer kan behlva uppdateras.
TODO Ta ut motionsmallen och skapa ett eget dokument för det.

gyback commented:
@eringh Kan du kolla på det här.
Jag vet inte riktigt hur vi ska göra med det sista.

Har du några synpunkter?

eringh commented:
Nu ligger du mil före mig i tanken. Behöver lite hjälp för att komma up to speed. Är detta relaterat till #1 och/eller #2 och/eller #3 ? Vad är det du vill typsätta? Hur vill du att det ska se ut och fungera?

Är tanken att slå ihop stil-mallen, med klassen för styrdokumenten? Och hur är dessa i så fall tänkta att användas? Sist jag va med och såg diskussionen verkade det finnas ett motstånd mot att gå tillbaka till TeX (även om jag ju gillar det).

gyback commented:
Den är relaterad till #1. Jag har utfört den merge som jag berättar om.

De flesta officiella dokumenten vi har är skrivna i TeX så det vore trevligt att få med styrdokumenten i samma stil. Det finns just nu inget större motstånd mot det och om fstildok har med allt som ser jobbigt ut så blir det enklare att lära sig.

Som jag förstår det så var du den senaste som arbetade med det här och jag undrar över vad syftet med Listingmiljö för paragrafer var och hur du tänkte att den skulle användas.

gyback committed:
Fixade typos och i Motion

Fixade ett typo i fstildok.tex och uppdaterade motionsdelen. Jag har parallellt fixat motionsmallen.

eringh commented:
Det kan vara så att jag va den som senast jobba med det, men @tlycken va nog den som utvecklade det.

En gång i tiden så va stadgarna skrivna i TeX och då va listmiljön för paragrafer använd för att få det snyggt uppstolkpat i paragrafer. Vet inte om det besvarar din fråga.

Gällande `allapm` så va det också sedan tidigare. Då var traditionen att varje PM va sitt eget dokument. Men det va ganska opraktiskt, varför det utvecklades funktionallitet för att få in alla PM i ett dokument, dock med numrering, titlesida, sidhuvud, m.m. anpassat som att det va olika PM ihopsatta. Detta utvecklades till att alla PM till slut bara hanterades som ett dokument, och har nu om jag förstått det rätt gått så långt att PM är ute och blivit ersatta av ett reglemente.
